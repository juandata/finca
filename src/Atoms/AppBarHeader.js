import React from 'react';
import { makeStyles } from '@material-ui/core/styles';
import AppBar from '@material-ui/core/AppBar';
import Toolbar from '@material-ui/core/Toolbar';
import Typography from '@material-ui/core/Typography';
import Logo from '../Assets/logoDiccionarioEmberaEspanol.jpeg'

const useStyles = makeStyles((theme) => ({
  root: {
    flexGrow: 1,
  },
  menuButton: {
    marginRight: theme.spacing(2),
  },
  title: {
    flexGrow: 1,
  },
  logoStyles: {
    width: '60px',
    marginRight: '10px'
  }
}));
/**
 * Encabezado, es la sección que renderiza el logo y el título de la app
 */
export default function AppBarHeader() {
  const classes = useStyles();
  return (
    <div className={classes.root}>
      <AppBar position="static">
        <Toolbar>
          {/*<img className={classes.logoStyles} src={Logo} alt='Logo Diccionario Español-Embera' />*/}
          <Typography variant="h6" className={classes.title}>
            Administrador Finca
          </Typography>

        </Toolbar>
      </AppBar>
    </div>
  );
}
// <Button color="inherit">Login</Button>