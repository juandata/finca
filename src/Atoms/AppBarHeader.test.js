import React from 'react';
import { shallow } from 'enzyme';
import AppBarHeader from './AppBarHeader';
describe('MyComponent', () => {
  it('should render correctly in "debug" mode', () => {
    const component = shallow(<AppBarHeader debug />);
  
    expect(component).toMatchSnapshot();
  });
});