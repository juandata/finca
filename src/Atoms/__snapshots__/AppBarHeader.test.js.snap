// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyComponent should render correctly in "debug" mode 1`] = `
<div
  className="makeStyles-root-1"
>
  <WithStyles(ForwardRef(AppBar))
    position="static"
  >
    <WithStyles(ForwardRef(Toolbar))>
      <img
        alt="Logo Diccionario Español-Embera"
        className="makeStyles-logoStyles-4"
        src="logoDiccionarioEmberaEspanol.jpeg"
      />
      <WithStyles(ForwardRef(Typography))
        className="makeStyles-title-3"
        variant="h6"
      >
        Diccionario Español-Embera / Embera-Español
      </WithStyles(ForwardRef(Typography))>
    </WithStyles(ForwardRef(Toolbar))>
  </WithStyles(ForwardRef(AppBar))>
</div>
`;
